ngshare/ngshare.py

```diff
@@ -898,6 +907,10 @@ def __init__(
         admin=(),
         autoreload=True,
     ):
+
+        if prefix.startswith('/healthz/'):
```

Comment:
Not related to this PR, but should we check `prefix.endswith('/')`?

Reply:
Maybe? That's frankly a good idea and should save some headaches related to weird 404s if the user specifies a custom prefix